src/realm/object-store/sync/sync_session.cpp

```diff
@@ -1521,6 +1522,8 @@ SyncProgressNotifier::NotifierPackage::create_invocation(Progress const& current
 {
     uint64_t transferred = is_download ? current_progress.downloaded : current_progress.uploaded;
     uint64_t transferrable = is_download ? current_progress.downloadable : current_progress.uploadable;
+    double progress_estimate =
```

Comment:
How come this value is 1.0 when `is_streaming` is true?

Reply:
I'm don't think this is the case here. I was actually trying our pre-existing tests (with PBS), and adding some conditions on the new progress estimate. And with streaming the estimate is always 1.0, while without it it gets a more reasonable value.